tests/acceptance/test_author_create_typehead.py

```diff
@@ -31,30 +31,31 @@
 def test_institutions_typehead(selenium, login):
     """"Test the suggestions in the istitutions field"""
     selenium.get(os.environ['SERVER_NAME'] + '/submit/author/create')
-    institution_field = selenium.find_element_by_id("institution_history-0-name")
-    force_autocomplete_event(institution_field, "CER")
-    assert 'CERN' in institution_field.get_attribute("value")
+    field = selenium.find_element_by_id("institution_history-0-name")
+    field.send_keys("CER")
+    sleep(5)
+    field.send_keys(Keys.DOWN)
+    field.send_keys(Keys.ENTER)
+    assert 'CERN' in field.get_attribute("value")
 
 
 def test_experiments_typehead(selenium, login):
     """"Test the suggestions in the experiments field"""
     selenium.get(os.environ['SERVER_NAME'] + '/submit/author/create')
-    experiment_field = selenium.find_element_by_id("experiments-0-name")
-    force_autocomplete_event(experiment_field, "ATL")
-    assert 'ATLAS' in experiment_field.get_attribute("value")
+    field = selenium.find_element_by_id("experiments-0-name")
+    field.send_keys("ATL")
+    sleep(5)
```

Comment:
Have you already tried using explicit waits here? Seems to be the recommended way: http://stackoverflow.com/questions/30986604/is-it-best-practice-to-use-thread-sleep-or-explicit-wait-before-click-on-any-e